rpm/qtbase.spec

```diff
@@ -543,6 +543,11 @@ install -D -p -m 0644 %{_sourcedir}/macros.qt5-default \
 mkdir -p %{buildroot}/etc/xdg/qtchooser
 ln -s %{_sysconfdir}/xdg/qtchooser/5.conf %{buildroot}%{_sysconfdir}/xdg/qtchooser/default.conf
 
+# Help accelerated qmake find the configuration
+%ifarch aarch64
```

Comment:
possible to test for word size in general?

Reply:
Changed to check the libdir, which is what we really want